pkg/etcd/etcd.go

```diff
@@ -0,0 +1,80 @@
+// etcd package is used for checking whether etcd disruption is allowd
```

Comment:
package comments should start with "Package <package name>"
https://tip.golang.org/doc/comment